Monika After Story/game/script-topics.rpy

```diff
@@ -4344,6 +4344,43 @@ label monika_confidence_2:
     m "Your everlasting love and care is just about all the support I need in order get to where I want to be."
     m "What kind of girlfriend would I be if I didn't return the favor~?"
     return
+    
+init 5 python:
+for key in ['book','tried','try','literature','original','originality']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('monika_book')
+    monika_random_topics.append('monika_book')
+
+label monika_book:
+    m "Despite the club usually being us exchanging poems, i've tried to write other things before."
+    m 1j "I've tried to write a book."
+    m 2e "It didn't work, though."
+    m 3m "I thought my idea was original, but it wasn't."
+    m 3n "I then found out that it was already done."
+    m 1q "It was about a spider helping a pig thats going to die."
+    m 1n "Sound familiar...?"
+    m 1l "Ehe..."
+    m 1q "I felt pretty down when I found out."
+    m 1h "But it's okay."
+    m 2e "Sometimes you'll find out that your ideas aren't totally original."
+    m 1r "But...is anything ever truly original?"
+    m 2i "Have you heard of that show...Moon and the Forces of Good?"
+    m 2l "Is that what it's called?"
+         menu:
```

Comment:
Remove one indent from the `menu` block (and code within the menu block). Here is a quick guide on indentation:

```
label this_is_label:
    m "i am dialogue"
    menu:
        m "I have question"
        "choice":
            m " I see you made choice"
        "other choice":
            m "I see you made other choice"
    m "more dialogue after choice"
    return
```